plugins/katello/nightly/installation/clients.md

```diff
@@ -26,42 +26,50 @@ Install the appropriate Katello client release packages.  For CentOS 6, you will
+yum install -y https://fedorapeople.org/groups/katello/releases/yum/{{ page.version }}/client/el5/x86_64/katello-client-repos-latest.rpm
+yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-5.noarch.rpm
 {% endhighlight %}
 </div>
 
 <div id="el6" style="display:none;" markdown="1">
 {% highlight bash %}
 wget https://copr.fedoraproject.org/coprs/dgoodwin/subscription-manager/repo/epel-6/dgoodwin-subscription-manager-epel-6.repo -O /etc/yum.repos.d/dgoodwin-subscription-manager-epel-6.repo
-yum -y install http://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
-yum install -y http://fedorapeople.org/groups/katello/releases/yum/{{ page.version }}/client/el6/x86_64/katello-client-repos-latest.rpm
+yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
+yum install -y https://fedorapeople.org/groups/katello/releases/yum/{{ page.version }}/client/el6/x86_64/katello-client-repos-latest.rpm
 {% endhighlight %}
 </div>
 
 <div id="el7" style="display:none;" markdown="1">
 {% highlight bash %}
-yum install -y http://fedorapeople.org/groups/katello/releases/yum/{{ page.version }}/client/el7/x86_64/katello-client-repos-latest.rpm
-yum -y install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+yum install -y https://fedorapeople.org/groups/katello/releases/yum/{{ page.version }}/client/el7/x86_64/katello-client-repos-latest.rpm
+yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
 {% endhighlight %}
 </div>
 
 <div id="f22" style="display:none;" markdown="1">
 {% highlight bash %}
-yum install -y http://fedorapeople.org/groups/katello/releases/yum/{{ page.version }}/client/f22/x86_64/katello-client-repos-latest.rpm
+yum install -y https://fedorapeople.org/groups/katello/releases/yum/{{ page.version }}/client/f22/x86_64/katello-client-repos-latest.rpm
 {% endhighlight %}
 </div>
 
 <div id="f23" style="display:none;" markdown="1">
 {% highlight bash %}
-yum install -y http://fedorapeople.org/groups/katello/releases/yum/{{ page.version }}/client/f23/x86_64/katello-client-repos-latest.rpm
+yum install -y https://fedorapeople.org/groups/katello/releases/yum/{{ page.version }}/client/f23/x86_64/katello-client-repos-latest.rpm
 {% endhighlight %}
 </div>
 
-Now you are ready to install the katello-agent:
+Now you are ready to install the client package:
+
+The `katello-host-tools` package reports errata, tracer & package profile information, but does not allow you to run remote actions on the clients. We generally recommend using Foreman Remote Execution or Ansible for remote actions.
+
+{% highlight bash %}
+yum install katello-host-tools -y
+{% endhighlight %}
+
+The agent package creates a Qpidd messaging bus between the client and the Server which allows you to run remote actions on many clients, this is often referred to as `via Katello agent`. This architecture does have some limitations when used with a large number of clients.
```

Comment:
> We generally recommend using Foreman Remote Execution or Ansible for remote actions.

I think this should be after the `yum install katello-host-tools`, and it can lead into the katello-agent part (i.e.):

```
We generally recommend using Foreman Remote Execution or Ansible for remote actions, but we also offer a messaging based client named katello-agent. The agent package ... 
```